packages/material-ui/src/MenuList/MenuList.js

```diff
@@ -6,66 +6,95 @@ import ReactDOM from 'react-dom';
 import warning from 'warning';
 import ownerDocument from '../utils/ownerDocument';
 import List from '../List';
+import getScrollbarSize from 'dom-helpers/util/scrollbarSize';
+
+function MenuList({ actions, children, className, onBlur, onKeyDown, disableListWrap, ...other }) {
+  const [currentTabIndex, setCurrentTabIndex] = React.useState(null);
+  const blurTimerRef = React.useRef();
+  const listRef = React.useRef();
+  const selectedItemRef = React.useRef();
+
+  React.useImperativeHandle(actions, () => ({
+    focus: () => {
+      if (selectedItemRef.current) {
+        ReactDOM.findDOMNode(selectedItemRef.current).focus();
```

Comment:
```suggestion
        selectedItemRef.current.focus();
```
`findDOMNOde` should be redundant here. This is never passed as-is to `ref` but already holds the returnvalue from `findDOMNode`.